src/cli/utils.js

```diff
@@ -298,19 +298,71 @@ export function getSection(search, field, store) {
  */
 export function matchKssDir(filepath, huron) {
   const kssSource = huron.get('kss');
-  /* eslint-disable space-unary-ops */
   // Include forward slash in our test to make sure we're matchin a directory, not a file extension
   const kssMatch = kssSource.filter((dir) => filepath.includes(`/${dir}`));
-  /* eslint-enable space-unary-ops */
 
   if (kssMatch.length) {
     return kssMatch[0];
   }
 
-  console.error(
-    chalk.red(`filepath ${filepath} does not exist in any
-    of the configured KSS directories`)
-  );
-
   return false;
 }
+
+/**
+ * Merge JSON files for css modules classnames in a provided directory
+ *
+ * @function mergeClassnameJSON
+ * @param {string} directory - directory containing classname JSON files
+ *
+ * @return {object} classnamesMerged - merged classnames. contents of each JSON file is nested within
+ *                           the returned object by filename. (e.g. article.json -> { article: {...json contents}})
+ */
+export function mergeClassnameJSON(directory) {
+  let files;
+
+  // Try to read through classnames directory
+  try {
+    files = fs.readdirSync(directory);
```

Comment:
Is this the only time you assign `files`? Could this be a const here?

Reply:
`files` is used elsewhere in the function, so I don't want it scoped to the `try` block